src/main/java/com/google/sps/data/RoomState.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.data;
+
+public class RoomState {
+
+    public static DataUtil.video_state currentState;
```

Comment:
That's why the video state should be in their own class. the prefix is not needed and strange here.